src/components/installedAppsView/InstalledAppsView.js

```diff
@@ -414,14 +414,27 @@ class InstalledAppsView extends React.Component {
       this.handleLaunchClicked.bind(this)
     );
     const managedTooltip = 'Managed services are delivered as a hosted service and supported by Red Hat.';
-    // const selfManagedTooltip = 'Self-managed services are available for use, but not managed by Red Hat.';
+    const selfManagedTooltip = 'Self-managed services are available for use, but not managed by Red Hat.';
 
     return (
       <div>
         <div className="integr8ly-tutorial-dashboard-title pf-l-flex pf-u-py-sm">
           <span className="pf-l-flex pf-m-inline-flex">
-            <h2 className="pf-c-title pf-m-3xl pf-u-mt-sm pf-u-mb-sm">Managed services</h2>
-            <Tooltip position="top" content={<div>{managedTooltip}</div>}>
+            <h2 className="pf-c-title pf-m-3xl pf-u-mt-sm pf-u-mb-sm">
+              {window.OPENSHIFT_CONFIG && window.OPENSHIFT_CONFIG.installationType === 'workshop'
```

Comment:
@dimitraz I know this is still a wip, but wanted to see if we could make a small change here. The default value should be "Self-managed services" and then it should change to "Managed services" if the installationType = "workshop" or "managed". The Bu asked that we not change the heading for workshop anymore. 

The update I'm suggesting though would allow for a default state for the SolX that would then capture if the installationType had no value or if it was something else like "self-managed" which would be coming soon from the RHI project. Feel free to ping me for additional discussions. FYI @aidenkeating

Reply:
@tiffanynolan That makes sense to me, I'll go ahead and make that change :)